benchmarks/user/benchmark.py

```diff
@@ -412,6 +412,7 @@ def param_string(self, params):
             if configuration['mpi']:
                 devito_params['np'] = MPI.COMM_WORLD.size
                 devito_params['rank'] = MPI.COMM_WORLD.rank
+                devito_params['mpi_mode'] = configuration['mpi']
```

Comment:
I suppose we should have a corresponding dummy value in the else branch otherwise things might fail when parsing the filename, depending on whether MPI was used or not

Reply:
I thought that if MPI is not used, there is no need to 'pollute' the filename. That is because in fact it is not a standalone parameter but a sub-parameter.

Reply:
but based on what I wrote above -- does it work if you are **not** using MPI ?

also, nitpicking: can we simply call it "mpi" ?

Reply:
Yes, let's go to mpi. It worked without MPI. I tested my bench scripts and also the plotting. The issue that persists is how to pass to plot functionality the info of the file it needs to parse. It should be done in a different way than it is now. Now you have to do `mpirun -n 2` but two threads plot together the same pic.